check_pgactivity

```diff
@@ -7065,9 +7065,10 @@ PARAM_LOOP: foreach my $row (@rs) {
 
 =item B<sequences_exhausted> (7.4+)
 
-Check all sequences assigned to a column (the smallserial, serial and bigserial
-types), and raise an alarm if the column or sequences gets too close to the
-maximum value.
+Check all sequences and raise an alarm if the column or sequences gets too
+close to the maximum value. The maximum value is calculated from the maxvalue
+of the sequence or from the column type when the sequence is assigned to a
```

Comment:
Should be "owned" rather than "assigned"?

Reply:
done in 73caa656410f68e02c2dcbc4499b3bd3add77fa2